java/CustomMetrics/README.md

```diff
@@ -0,0 +1,37 @@
+# Custom Metrics
+
+* Flink version: 1.15.2
+* Flink API: DataStream API
+* Language: Java (11)
+
+
+This example demonstrate how to create your own metrics to track application-specific data, such as processing events or accessing external resources and publish it to Cloudwatch.
+
+The custom metrics is explained by two different application -
+**RecordCount** - This application shows how to create a custom record count metric and publish it to Cloudwatch.
```

Comment:
I think this was supposed to be a bulleted list

Reply:
done